documentation/importing-dependencies/tutorial-importing.asciidoc

```diff
@@ -36,68 +38,26 @@ The functionality is the same as for the annotations.
   }
 ----
 
-[NOTE]
-====
-For every class that imports dependencies, import order is guaranteed for dependencies of the same type only.
-No guarantees are given regarding numerous classes' dependencies' import order, same as no guarantees are given on import order of
-dependencies of not the same type.
+== Storing JavaScript and HTML Files
 
-Example:
-[source, java]
-----
-@JavaScript("1.js")
-@StyleSheet("1.css")
-@HtmlImport("1.html")
-@JavaScript("2.js")
-@StyleSheet("2.css")
-@HtmlImport("2.html")
-static class OrderedDependencies extends Div {
-}
-----
-In this case, `1.js` will be imported before `2.js`, `1.css` before `2.css`, `1.html` before `2.html`, but no other guarantees can be made.
+// This content is repeated on the Including CSS page. Should we combine these pages?
```

Comment:
Let's remove this, duplicate content. Should be only in tutorial-ways-of-importing.asciidoc and we could refer to that from here.